google/resource_compute_instance.go

```diff
@@ -1056,10 +1120,8 @@ func resourceComputeInstanceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("disk", disks)
 	d.Set("attached_disk", attachedDisks)
 	d.Set("scratch_disk", scratchDisks)
-
-	scheduling := flattenScheduling(instance.Scheduling)
-	d.Set("scheduling", scheduling)
-
+	d.Set("scheduling", flattenBetaScheduling(instance.Scheduling))
+	d.Set("guest_accelerator", flattenGuestAccelerators(instance.Zone, instance.GuestAccelerators))
```

Comment:
(Line below) If we don't convert the self link to v1, we will cause diffs in resources that refer to this one; you can use the existing function I was using, or move to a `StateFunc`.

In this case I'd probably do the former because it is already a big PR, and you can clean up all the old function usages at once in a separate PR when you have time.